reference/orders.v2.oas2.yml

```diff
@@ -4788,9 +4838,7 @@ components:
       description: |
        **Usage notes:**
 
-       To `add` a custom product to an existing order, don't include `id` in the payload. You must provide a nonempty value for at least one of these fields: `name`, `name_customer`, or `name_merchant`.
```

Comment:
why do we need to remove this line ?

Reply:
ah got it, we have `orderCustomProduct_Post` as well, thanks for cleaning this up 🙇